src/Common/src/CoreLib/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -10,7 +10,10 @@ internal static class HashHelpers
 
         public static int Combine(int h1, int h2)
         {
-            return ((int)BitOps.RotateLeft((uint)h1, 5) + h1) ^ h2;
+            // RyuJIT optimizes this to use the ROL instruction
```

Comment:
This can be changed back to RotateLeft once `BitOpts` become proper public API.

Reply:
Thanks @jkotas

Reply:
> once BitOpts become proper public API

@jkotas can I go ahead with this (make it public). There is a bunch of callsites in `CoreFX`, `CoreRT` and externally that I want to update, and I don't want to have do tricks like this to get them working.

Reply:
Only once there is approved public API shape. Once the APIs get used in CoreFX, it is pain to change them.

Reply:
Could you please update https://github.com/dotnet/corefx/issues/32269 with the list of APIs that we have today?

I would omit the get/set functions from the first wave of the API because of I expect it would take a while to get to converge on them.